docs/dashboard.md

```diff
@@ -2,8 +2,12 @@
 
 Using dashboard you can monitor SC4SNMP and be sure that is healthy and working correctly.
 
+
 ## Presetting
 
+!!! info
+    Dashboard compatible with SC4SNMP 1.12+
```

Comment:
Add this disclaimer, because in 1.11 we not provided port number in IP

Reply:
Is !!! some markdown? If not I think we can replace it with:
**Info
    Dashboard compatible with SC4SNMP 1.12+**

Reply:
<img width="1728" alt="Screenshot 2024-10-09 at 09 03 03" src="https://github.com/user-attachments/assets/ba999fe9-e5aa-4010-865e-aac445427588">